src/main/kotlin/com/hibob/ex5_classes_exercise/Meeting.kt

```diff
@@ -0,0 +1,32 @@
+package com.hibob.ex5_classes_exercise
+
+interface Location{}
+
+data class Us(val zipCode: Int) : Location {}
+
+data class Uk(val postalCode: Int) : Location {}
+
+data class LocationDetails(val street: String, val city: String, val county: String, val loc:Location) {}
```

Comment:
why not adding all of this to the location interface?

Reply:
updated, thanks!